openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx

```diff
@@ -240,44 +350,77 @@ const ExplorePage: FunctionComponent = () => {
       })
       .finally(() => setIsLoading(false));
   }, [
-    searchIndex,
+    parsedSearch.quickFilter,
+    queryFilter,
     searchQueryParam,
     sortValue,
     sortOrder,
     showDeleted,
-    advancesSearchQueryFilter,
     elasticsearchQueryFilter,
-    queryFilter,
+    searchIndex,
     page,
   ]);
 
-  const handleAdvanceSearchQueryFilterChange = useCallback(
-    (filter?: Record<string, unknown>) => {
+  const handleAdvanceSearchQuickFiltersChange = useCallback(
+    (filter?: QueryFilterInterface) => {
       handlePageChange(1);
-      setAdvancedSearchQueryFilter(filter);
+      setAdvancedSearchQuickFilters(filter);
+      handleQuickFilterChange(filter);
     },
-    [setAdvancedSearchQueryFilter]
+    [setAdvancedSearchQuickFilters, history, parsedSearch]
   );
 
   useEffect(() => {
     AppState.updateExplorePageTab(tab);
   }, [tab]);
 
+  useEffect(() => {
+    try {
+      const newAggregates: Aggregations = {};
+
+      if (
+        !isEmpty(withFilterAggregations) &&
+        !isEmpty(withoutFilterAggregations) &&
+        !isUndefined(withoutFilterAggregations) &&
+        !isUndefined(withFilterAggregations)
+      ) {
+        Object.keys(withoutFilterAggregations).forEach((filterKey) => {
+          const aggregateFieldValue = getUpdatedAggregateFieldValue(
```

Comment:
Is this a API call? If yes we need to rewrite this in a batter way

Reply:
No its a utility function.